paper/paper.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "Pooch: A friend to fetch your data files"
+tags:
+  - python
+authors:
+  - name: Leonardo Uieda
+    orcid: 0000-0001-6123-9515
+    affiliation: 1
+  - name: Santiago Rubén Soler
+    orcid: 0000-0001-9202-5317
+    affiliation: "2,3"
+  - name: Rémi Rampin
+    orcid: 0000-0002-0524-2282
+    affiliation: 4
+  - name: Hugo van Kemenade
+    orcid: 0000-0001-5715-8632
+    affiliation: 5
+  - name: Matthew Turk
+    orcid: 0000-0002-5294-0198
+    affiliation: 6
+  - name: Daniel Shapero
+    orcid: 0000-0002-3651-0649
+    affiliation: 7
+affiliations:
+ - name: Department of Earth, Ocean and Ecological Sciences, School of Environmental Sciences, University of Liverpool, UK
+   index: 1
+ - name: Instituto Geofísico Sismológico Volponi, Universidad Nacional de San Juan, Argentina
+   index: 2
+ - name: CONICET, Argentina
+   index: 3
+ - name: New York University, USA
+   index: 4
+ - name: Independent (Non-affiliated)
+   index: 5
+ - name: University of Illinois at Urbana-Champaign, USA
+   index: 6
+ - name: Polar Science Center, University of Washington Applied Physics Lab, USA
+   index: 7
+date: 23 October 2019
+bibliography: paper.bib
+---
+
+# Summary
+
+Scientific software are usually created to analyze, model, and visualize data.
```

Comment:
software is uncountable
```suggestion
Scientific software is usually created to analyze, model, and visualize data.
```
or maybe "scientific software packages/products/projects"